### PR TITLE
feature/#27-new-finding-azure-keyvault-no-network-acl

### PR DESCRIPTION
**Pull request by misg**
This PR adds support for the Key Vault service.
Please look at the [proprietary PR](https://github.com/nccgroup/ScoutSuite-Proprietary/pull/75) as well.

### CODE REVIEW

File: ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+</script>
+
+<script>
+    Handlebars.registerPartial("services.keyvault.vaults", $("#services\\.keyvault\\.vaults\\.partial").html());
+</script>
+
+<!-- Single Key Vault template -->
+<script id="single_keyvault_vaults-template" type="text/x-handlebars-template">
+    <div style="text-align: right; padding-right: 10px; text-weight: bold;"><a href="javascript:hidePopup()">X</a></div>
+    {{> services.keyvault.vaults}}
```
Comment (Aboisier) [inline ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html]: I'm sorry what

Comment (Aboisier) [reply ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html]: Allow me to remind you of this [pain in the rear](https://github.com/nccgroup/ScoutSuite/pull/132/files#diff-795b3bcacbbb37ba1939693c79cb20a7) of a pull request.

Comment (misg) [reply ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html]: Nice catch, wrong copy/paste... Sorry about that. :x

Comment (Aboisier) [reply ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html]: Hahaha, I accept yout apology.

Comment (Aboisier) [reply ScoutSuite/output/data/html/partials/azure/services.keyvault.vaults.html]: Hahaha, I accept yout apology.

**Pull request merged**